docs/docs.html

```diff
@@ -382,6 +382,26 @@ <h1 class="heading">REGAL UI</h1>
             class="code-snippet"
             sandbox="allow-scripts allow-same-origin">
             </iframe> <hr class="mb-3 mt-3" />
+
+            <!-- Snackbar -->
+            <div id="rating" class="heading1 bolder mt-5 mb-5">Rating</div>
+            <div class="overview">Ratings provide insight regarding others' opinions and experiences, and can allow the user to submit a rating of their own.</div>
+            <div class="heading2 mt-1 mb-1">User Rating</div> <br>
+            <div class="wrap mb-3">
+                <span class="fa fa-star checked"></span>
+                <span class="fa fa-star checked"></span>
+                <span class="fa fa-star checked"></span>
+                <span class="fa fa-star "></span>
+                <span class="fa fa-star"></span>
+            </div>
+            <p class="headin4">3 average based on 300 reviews.</p> <br>
```

Comment:
Is the spelling correct here? "headin". I think it is a type.

Reply:
yeah, its typo error Thanks!!